src/apiClient.ts

```diff
@@ -187,6 +186,10 @@ export default class APIClient {
 
       return response;
     } catch (error) {
+      if (error instanceof Error && error.name === "AbortError") {
```

Comment:
@mmollick is it possible for you to either run prettier (via `npm run lint:prettier`) or just use single quotes here so we can get the CI check to pass, it would be greatly appreciated!

```suggestion
      if (error instanceof Error && error.name === 'AbortError') {
```

Reply:
Done